docs/getting-started.md

```diff
@@ -1,18 +1,13 @@
 # Getting started
 
-## 1. Get your project added to the config preview
```

Comment:
We should probably keep a heading here but change the instructions to be to turn on Build Processing under Advanced on your project's settings page.